include/gazebo_motor_model.h

```diff
@@ -111,6 +116,11 @@ class GazeboMotorModel : public MotorModel, public ModelPlugin {
   int motor_number_;
   int turning_direction_;
 
+  int motor_Failure_Number_; /*!< motor_Failure_Number is (motor_number_ + 1) as (0) is considered no_fail. Publish accordingly */
+  int tmp_motor_num;
```

Comment:
Same thing as above - it already is on `gazebo_motor_failure_plugin.h` header. Also please make it `bool` type instead and add a comment on front so to document what it does.